src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -331,9 +332,9 @@ protected function instantiateObject(array &$data, string $class, array &$contex
                 return $reflectionClass->newInstanceWithoutConstructor();
             }
 
+            $context['missing_constructor_arguments'] = [];
```

Comment:
Why is it set in the context here? Can't it be a variable?

Reply:
I made it like this to keep the same "code philosophy" as for `not_normalizable_value_exceptions` on line 401.

Reply:
ok, I supposed it was some kind of "convention". I've fixed that.